app/assets/javascripts/admin/addon/components/watched-word-uploader.js

```diff
@@ -3,7 +3,7 @@ import I18n from "I18n";
 import UppyUploadMixin from "discourse/mixins/uppy-upload";
 import { alias } from "@ember/object/computed";
 import bootbox from "bootbox";
-import discourseComputed from "discourse-common/utils/decorators";
+import { observes } from "discourse-common/utils/decorators";
```

Comment:
We're trying to reduce our usage of observers because of all their indirections. I _think_ we usually replace them with proper actions / event handling.